src/mumble/widgets/EventFilters.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_WIDGETS_EVENTFILTERS_H_
+#define MUMBLE_MUMBLE_WIDGETS_EVENTFILTERS_H_
+
+#include <initializer_list>
+#include <vector>
+
+#include <QEvent>
+#include <QObject>
+#include <QPoint>
+
+class KeyEventObserver : public QObject {
+	Q_OBJECT
+
+private:
+	QEvent::Type m_eventType;
+	bool m_consume;
+	std::vector< Qt::Key > m_keys;
+
+public:
+	KeyEventObserver(QObject *parent, QEvent::Type eventType, bool consume, std::initializer_list< Qt::Key > keys);
```

Comment:
Take the argument as a vector. That can implicitly be constructed from a suitable initializer list. And that we we don't restrict ourselves to only constructing these things from initializer lists.
(take vector by value and `move` it in the ctor impl)

Then you also don't need the `initializer_list` header anymore